doc/2/controllers/security/delete-profile/snippets/delete-profile.dart

```diff
@@ -0,0 +1,11 @@
+final result = await kuzzle.security.deleteProfile('myProfile');
+print('Profile successfully deleted');
```

Comment:
Please do the print in the template by using the template `default` or `print-result` for example.